frontend/src/main/pages/AdminJobsPage.js

```diff
@@ -71,16 +56,24 @@ const AdminJobsPage = () => {
                         <TestJobForm submitAction={submitTestJob} />
                     </Accordion.Body>
                 </Accordion.Item>
-                {
-                    jobLaunchers.map((jobLauncher, index) => (
-                        <Accordion.Item eventKey={index + 1}>
-                            <Accordion.Header>{jobLauncher.name}</Accordion.Header>
-                            <Accordion.Body>
-                                {jobLauncher.form}
-                            </Accordion.Body>
-                        </Accordion.Item>
-                    ))
-                }
+               <Accordion.Item eventKey="1">
+                   <Accordion.Header>Milk the Cows</Accordion.Header>
+                   <Accordion.Body>
+                       <JobComingSoon/>
+                   </Accordion.Body>
+               </Accordion.Item>
+               <Accordion.Item eventKey="2">
+                   <Accordion.Header>Instructor Report</Accordion.Header>
+                   <Accordion.Body>
+                       <InstructorReportJobForm submitAction={submitTestJob} />
+                   </Accordion.Body>
+               </Accordion.Item>
+               <Accordion.Item eventKey="3">
+                   <Accordion.Header>Update Cow Health</Accordion.Header>
+                   <Accordion.Body>
+                       <UpdateCowHealthJobForm submitAction={submitTestJob} />
```

Comment:
You need a submit action for an Update Cow Health Job.

You cannot use submitTestJob here, since it submits the wrong job!